pkg/kritis/testutil/util.go

```diff
@@ -83,14 +77,22 @@ func getKey(key *openpgp.Entity, keyType string, t *testing.T) []byte {
 		CheckError(t, false, key.Serialize(wr))
 	}
 	wr.Close()
-	return gotWriter.Bytes()
+	return string(gotWriter.Bytes())
```

Comment:
I believe that you should be able to skip this conversion by using: return gotWriter.String()

Ref: https://golang.org/pkg/bytes/#Buffer.String